examples/geodesic_regression_hypersphere.py

```diff
@@ -73,8 +73,8 @@ def main():
     tangent_vec_of_transport = METRIC.log(intercept, base_point=intercept_hat)
     transported_coef_hat = METRIC.parallel_transport(
         tangent_vec_a=tangent_vec_to_transport,
```

Comment:
Should we rename tangent_vec_a and tangent_vec_b : they play different roles, and I always have to go back to the file implementing the parallel transport to remember which one is which one.

How about: `tangent_vec_to_transport` and `tangent_vec`?

Reply:
what about `tangent_vec` and `initial_direction`?

Reply:
or tangent_vec and initial_tangent_vec? which ever you prefer!